datahub-web-react/src/app/entity/shared/tabs/Dataset/Stats/StatsTab.tsx

```diff
@@ -58,15 +58,19 @@ export default function StatsTab() {
 
     const latestStats = (
         <>
-            <TableStats
-                rowCount={latestProfile?.rowCount || undefined}
-                columnCount={latestProfile?.columnCount || undefined}
-                queryCount={usageStats?.aggregations?.totalSqlQueries || undefined}
-                users={usageStats?.aggregations?.users || undefined}
-                lastUpdatedTime={lastUpdatedTime || undefined}
-                lastReportedTime={lastReportedTime || undefined}
-            />
-            <ColumnStats columnStats={(latestProfile && latestProfile.fieldProfiles) || []} />
+            {(latestProfile?.rowCount || latestProfile?.columnCount || usageStats?.aggregations?.totalSqlQueries) && (
```

Comment:
What if there are none of these things, but there is a set of top users, or a last updated time. We just hide all of this Can we solve it inside the TableStats component?

Reply:
Removed the condition and handled it in TableStats.tsx file. 

`if (
        !rowCount &&
        !columnCount &&
        !queryCount &&
        !(users && users.length > 0) &&
        !lastUpdatedTime &&
        !lastReportedTime
    ) {
        return null;
    }`